runtime/tutor

```diff
@@ -988,6 +988,28 @@ lines.
 
 
 
+=================================================================
+=             9.4 JUMP WITH TWO-CHARACTER LABELS                =
+=================================================================
+
+ Type gw to enable the 2-character labels. The start of each word
+ will be replaced by 2 highlighted characters. Type any sequence
+ of 2 highlighted characters to jump to the corresponding label,
+ or use ESC to drop the labels.
+
+ The 2-character labels allow to quickly jump to any location
+ in the viewable selection.
+
+ 1. Move the cursor to the start of the line marked '-->' below.
+ 2. Press gw to enable the 2-character labels, and then bm to
+    jump to the word here (which appears as bmre with the label).
```

Comment:
Which characters are shown depends on how you have your jump alphabet configured. It's probably unlikely you'll be reading this section if you've configured your jump alphabet but I think we can avoid it by saying something like "press the two characters that appear at the start of "here""